nlp_architect/procedures/token_tagging.py

```diff
@@ -143,6 +143,14 @@ def add_parse_args(parser: argparse.ArgumentParser):
                         help="Overwrite the content of the output directory")
     parser.add_argument("--no_cuda", action='store_true',
                         help="Avoid using CUDA when available")
+    parser.add_argument('--log_file', type=str, default="log_file",
+                            help='log path for evaluation output')
+    parser.add_argument("--bilou", action='store_true',
+                        help="whether to use bilou format evaluation")
+    parser.add_argument('--drop_penalty', type=float, default=1e-4,
+                        help='idcnn dropout penalty')
+    parser.add_argument('--ignore_token', type=str, default="",
+                        help='a token to ignore when processing the data')
```

Comment:
are these new arguments supported by all the taggers?

Reply:
yes